README.md

```diff
@@ -15,6 +15,23 @@ ember install ember-frost-info-bar
 ```
 
 ## Examples
+### Block Format
+```handlebars
+{{#frost-info-bar as |slot|}}
+  {{#block-slot slot 'title'}}
+      <div class='primary-title'>User accounts</div>
```

Comment:
I think we should also add a slot for 'context-controls' which would be a location for tabs, search fields, checkboxes, etc.  This would be an optional area centered on the page between the title/summary and actions.

```handlebars
{{#block-slot slot 'context-controls'}}
  ...
{{/block-slot}}
```